tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1385,10 +1385,11 @@ class CompletionSuite extends BaseCompletionSuite {
     "select-ignores-next-line",
```

Comment:
Checking `CompletionsBaseSuite` made me realize that this test was never checked and didn't work on Scala 2